### DISCUSSION TITLE
[Dialog] Scrollbar does not respond to clicks

### DISCUSSION THREAD

**Discussion by Xelson** [chakra-ui/zag#2027] [General]
# 🐛 Bug report

Scrollbar does not respond to clicks inside `dialog`

## 💥 Steps to reproduce

1. Go to reproduction sandbox
2. Click on `open dialog` button
3. See that the scrollbar does not respond to clicks

## 💻 Link to reproduction

CodeSandbox reproduction: https://stackblitz.com/edit/vitejs-vite-ybc2ej

## 🌍 System information

| Software         | Version(s) |
| ---------------- | ---------- |
| Zag Version      |     0.77.1      |
| Browser          |     Brave       |
| Operating System |   Windows         |

**Comment by segunadebayo** [comment]
This isn’t a bug. It’s related to how you styled the dialog. 

Will share a better approach when I have time
